### ISSUE TITLE
adaptive card images loading from https://urlp.sfbassets.com

### ISSUE THREAD

**Issue by liangchr** [MicrosoftDocs/msteams-docs#1946]
Could we have adaptive card images load from a teams.microsoft.com domain? (not https://urlp.sfbassets.com) Some users proxies enabled that prevent loading from anything other than teams.microsoft.com. Our apps images are not showing up on adaptive cards sent into the chat.

**Comment by Nikitha-MSFT** [comment]
No, You cannot get images from teams.microsoft.com. Also the images which you have given in the adaptivecard should be publicly available. For more information could you please check this [docs](https://docs.microsoft.com/en-us/microsoftteams/platform/task-modules-and-cards/cards/cards-reference#inline-card-images)?
